### PR TITLE
[Task] Feature Detail Update images

### PR DESCRIPTION
**Pull request by ErickDoroy**
## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202381121175300/f
- Route Link: http://localhost:3000/feature/adopt

## Description

- Update new image in Feature Details
- Make image to WEBP format

## Notes

N/A

## Screenshots

image

image

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
     "eslint-config-airbnb": "^19.0.4",
     "i18next": "^21.8.4",
     "i18next-browser-languagedetector": "^6.1.4",
-    "eslint-config-prettier": "^8.5.0",
```
Comment (roseaugusto) [inline package.json]: @ErickDoroy Why is this removed?

File: yarn.lock [hunk 2]
```diff
@@ -2398,7 +2398,12 @@
   dependencies:
     "ajv" "^8.0.0"
 
-"ajv-keywords@^3.4.1", "ajv-keywords@^3.5.2":
+"ajv-keywords@^3.4.1":
```
Comment (roseaugusto) [inline yarn.lock]: yarn.lock and package.lock should be excluded if no packages are added. Please remove this.

**Review by roseaugusto — APPROVED** [review]
Approved!

**Review by egiegarciano14 — APPROVED** [review]
